src/components/Lesson/ImprovePage.js

```diff
@@ -25,20 +24,20 @@ const ImprovePage = React.createClass({
     return (
       <div className={styles.container}>
         <div className={
-          this.props.isStudentMode ? styles.improvePageBoxStudent : styles.improvePageBoxTeacher}>
+          this.props.isStudentMode ? styles.student : styles.teacher}>
```

Comment:
Hvis du vil gjøre det litt penere (ihvertfall hvis du skal ha en slik ternary flere ganger med `this.props.isStudentMode`), så kunne du jo definert `const isStudent = this.props.isStudentMode` før return...